apps/web/src/app/(signing)/sign/[token]/signing-field-container.tsx

```diff
@@ -60,19 +60,16 @@ export const SigningFieldContainer = ({
             'text-foreground hover:shadow-primary-foreground group flex h-full w-full flex-col items-center justify-center p-2',
           )}
         >
-          {!field.inserted && !loading && (
-            <button type="submit" className="absolute inset-0 z-10" onClick={onSignFieldClick} />
-          )}
-
-          {field.inserted && !loading && (
-            <button
-              className="text-destructive bg-background/40 absolute inset-0 z-10 flex items-center justify-center rounded-md text-sm opacity-0 backdrop-blur-sm duration-200 group-hover:opacity-100"
-              onClick={onRemoveSignedFieldClick}
-            >
-              Remove
-            </button>
-          )}
-
+          <button
```

Comment:
Same situation here, prefer to have the duplication to keep states easier to manage 😄

Reply:
Oki Doki. I will revert this as well. Could you please let me know about the other changes? If I should keep them or revert them along with the other two you suggested?

Reply:
Just those two for now

Reply:
Done 🚀

Reply:
Done 🚀